README.md

```diff
@@ -20,14 +22,29 @@ Or install it yourself as:
 
 ## Usage
 
+Scraped currently has support for working with HTML and JSON documents.
+
+To write a scraper, start by creating a subclass of `Scraped::HTML` or `Scraped::JSON` for each _type_ of page you’re scraping. Then specify the (data) fields you want to extract from that. You can control the strategy used to get the page, and decorate the response you get back to make it easier to parse.
```

Comment:
The emphasis on 'type' makes it seem important, so I think we need something to spell out a bit more what that means (whether inplace here, or a pointer to the section that goes into that)